pro.affine.AFFiNE.yaml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/refs/heads/main/data/flatpak-manifest.schema.json
+app-id: pro.affine.AFFiNE
```

Comment:
why is the last component mixed case?

Reply:
Copied naming from their github repo: https://github.com/toeverything/AFFiNE

Reply:
GitHub namespaces are case insensitive and it doesn't matter here.

Reply:
Don't have a preference either name, will rename it to all lower case